java/code/src/com/suse/manager/webui/controllers/FormulaController.java

```diff
@@ -113,7 +113,7 @@ public void initRoutes(JadeTemplateEngine jade) {
                 withCsrfToken(withUser(this::serverGroupFormula)),
                 jade);
         get("/manager/systems/details/formula/:formula_id",
-                withCsrfToken(this::minionFormula),
+                withCsrfToken(withUser(this::minionFormula)),
```

Comment:
Just a suggestion for improvement: Alternatively to the approach in this commit ("Localize jade file documentation links"), we could think of the following alteratives:

- instead of `withUser`, create `withDocsLocale` helper, that would inject the locale to the model (`data.put("docsLocale"...` could be skipped in the individual handlers
- or we could inject the `docsLocale` into the shared variables (`SparkApplicationHelper`), but I think this would be a bit more complicated (we couldn't inject just the value, since the `setup` method is only called once. instead of that, we would need to inject a method for computing that stuff). so maybe not :)

Using one of these would get rid of manual filling of the `docsLocale` model parameter. I'm up for a discussion about this!

Reply:
There is only one concern I have with this approach. When writing a controller method that has a jade file that uses docsLocale, you would have to add the `user` (to get the users docsLocale) to the method signature although the method itself might not need to use the user. This is somewhat counter intuitive.

But I agree, that it would make the code cleaner. So no strong opinion against it.

Reply:
Actually forget my last comment. It worked out differently than I thought it would. I didn't need to add the `user` to the methods signatures. Just check out the result and resolve if you're satisfied.

Reply:
LGTM! Thanks :+1: